internal/adapter/outbound/alkemiodb/reference_it_test.go

```
@@ -0,0 +1,265 @@
+		got = append(got, uuid.UUID(b))
+	}
+	if len(got) < 2 {
+		t.Skip("need at least two distinct storage buckets in the DB")
+	}
+	return got[0], got[1]
+}
+
+// T014: create → by-reference (global & scoped) → PATCH move (bucket + auth +
+// createdBy + ref) → by-reference resolves in the new bucket; the old bucket
+// misses; global still resolves.
+func TestByReference_MoveResolution(t *testing.T) {
```

golangci-lint (2.12.2)

[error] cyclomatic complexity 17 of func `TestByReference_MoveResolution` is high (> 15)

(gocyclo)